_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,894 @@
+- The [declarative schema changer](../{{site.versions["dev"]}}/online-schema-changes.html#declarative-schema-changer) now falls back to the legacy schema changer when a user-defined function is found in the dependency graph when encountering a `DROP` statement. This no longer throws an unimplemented error. [#85981][#85981]
+- Arrays can now be [imported](../{{site.versions["dev"]}}/import.html) in a CSV file using the `{}` format, similar to `COPY FROM`. Importing array expressions (e.g., `ARRAY[1, 2, 3]`) is still supported as well. [#85850][#85850]
+- Creating a not visible index using `CREATE TABLE …(INDEX … NOT VISIBLE)` or `CREATE INDEX … NOT VISIBLE` is now supported. [#85794][#85794]
+- The output from [`SHOW STATISTICS`](../{{site.versions["dev"]}}/show-statistics.html) is now more deterministic. [#77070][#77070]
+- Added a new `WITH FORECAST` option to the [`SHOW STATISTICS`](../{{site.versions["dev"]}}/show-statistics.html) statement, which calculates and displays forecasted statistics along with the existing table statistics. [#77070][#77070]
+- Added the `trunc(decimal, int)` [built-in function](../{{site.versions["dev"]}}/functions-and-operators.html), which truncates the given decimal value to the specified number of decimal places. A negative value can be used for the scale parameter, which will truncate to the left of the decimal point. Example: [#85890][#85890]
+
+    ~~~sql
+    SELECT trunc(541.234, 2), trunc(541.234, 0), trunc(541.234, -1);
+    trunc  | trunc | trunc
+    -------+-------+---------
+    541.23 |   541 | 5.4E+2.
+    ~~~
+
+- The `CREATE EXTERNAL CONNECTION` statement can be used to represent an underlying [`userfile`](../{{site.versions["dev"]}}/use-userfile-for-bulk-operations.html) resource. [#86006][#86006]
+- Altering an index to visible or not visible using `ALTER INDEX … VISIBLE` or `NOT VISIBLE` is now supported. [#86032][#86032]
+- When performed by the [declarative schema changer](../{{site.versions["dev"]}}/online-schema-changes.html#declarative-schema-changer) (as is the case by default) the [`ALTER PRIMARY KEY`](../{{site.versions["dev"]}}/alter-primary-key.html) statement now also drops the `rowid` column when no references are held to it anywhere. The `rowid` column is a hidden column which is implicitly added and serves as primary key on any table created without explicitly specifying a primary key. [#86071][#86071]
+- [Session setting](../{{site.versions["dev"]}}/set-vars.html) `optimizer_use_not_visible_indexes` and [cluster setting](../{{site.versions["dev"]}}/cluster-settings.html) `sql.defaults.optimizer_use_not_visible_indexes.enabled` can be used to disable not visible index features. When these settings are enabled, the optimizer treats not visible indexes as if they were visible and can choose to use them for query planning. By default, these settings are disabled. [#86033][#86033]
```

Comment:
According to https://github.com/cockroachdb/docs/issues/14819#issuecomment-1224296013, the cluster setting was removed.